tests/h/feeds/atom_test.py

```diff
@@ -1,13 +1,26 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=protected-access
 """Unit tests for h/atom.py."""
+from datetime import datetime
 import mock
 
+from h.api import models
 from h.feeds import atom
 
 from ... import factories
 
 
+def _annotation(**kwargs):
+    args = {
+        'userid': 'acct:janebloggs@hypothes.is',
+        'created': datetime.now(),
```

Comment:
To avoid confusion about what we expect datetimes in our system to be, I'd suggest _always_ using `utcnow()`, even in cases like this where it doesn't matter.